### PR TITLE
Add tutorial for modifying kernel boot parameters

### PR DESCRIPTION
**Pull request by intelkevinputnam**
Need to fill out 2 sections:

1. Example of adding a boot parameter
2. Example of restoring defaults

Would also be useful to offer a way to recover from getting into the situation where changing boot parameters has caused the system to not be able to boot.

### CODE REVIEW

**Comment by intelkevinputnam** [comment]
Addresses issue https://github.com/clearlinux/clear-linux-documentation/issues/475

File: source/clear-linux/tutorials/kernel-boot-params.rst [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+   Be careful when editing boot parameters as the system can be made
+   non-bootable.
+
+.. contents:: 
+    :local:
+    :depth: 1
+
+Add a boot parameter
+********************
+
```
Comment (bwarden) [inline source/clear-linux/tutorials/kernel-boot-params.rst]: sudo mkdir -p /etc/kernel/cmdline.d
echo "debug" | sudo tee /etc/kernel/cmdline.d/debug.conf

File: source/clear-linux/tutorials/kernel-boot-params.rst [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+consistent with its :ref:`stateless` nature. All configuration changes belong
+in the :file:`/etc/kernel/` path. The :command:`clr-boot-manager` generates
+boot entries, so we'll use it to run an update each time we change a boot
+paramter. 
+
+.. warning::
+
+   Be careful when editing boot parameters as the system can be made
+   non-bootable.
+
```
Comment (bwarden) [inline source/clear-linux/tutorials/kernel-boot-params.rst]: I'd suggest `clr-boot-manager get-timeout` and `set-timeout` to set a non-zero boot timeout before experimenting with changes. That way, if boot fails, they can try again, hit `e` at the boot menu to edit the command line, and delete the entries they added.

File: source/clear-linux/tutorials/kernel-boot-params.rst [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+Modify kernel boot parameters
+#############################
+
+Changing kernel boot paramters can drastically change the way the OS behaves,
+so be cautious when doing so. |CL| provides a simple way to do this that is
+consistent with its :ref:`stateless` nature. All configuration changes belong
+in the :file:`/etc/kernel/` path. The :command:`clr-boot-manager` generates
+boot entries, so we'll use it to run an update each time we change a boot
+paramter.
```
Comment (bwarden) [inline source/clear-linux/tutorials/kernel-boot-params.rst]:
```suggestion
parameter. 
```

File: source/clear-linux/tutorials/kernel-boot-params.rst [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+Modify kernel boot parameters
+#############################
+
+Changing kernel boot paramters can drastically change the way the OS behaves,
+so be cautious when doing so. |CL| provides a simple way to do this that is
+consistent with its :ref:`stateless` nature. All configuration changes belong
+in the :file:`/etc/kernel/` path. The :command:`clr-boot-manager` generates
+boot entries, so we'll use it to run an update each time we change a boot
+paramter. 
+
```
Comment (bwarden) [inline source/clear-linux/tutorials/kernel-boot-params.rst]: Documentation for the command line modification process is in the CONFIGURATION section of `man clr-boot-manager`

**Comment by intelkevinputnam** [comment]
@bwarden Thanks! I'll incorporate your feedback into the doc.

**Comment by intelkevinputnam** [comment]
@bwarden I'm assuming deleting the contents of /etc/kernel/cmdline.d and /etc/cmdline-removal.d and running sudo clr-boot-manager update would reset default. Correct?

**Comment by bwarden** [comment]
> @bwarden I'm assuming deleting the contents of /etc/kernel/cmdline.d and /etc/cmdline-removal.d and running sudo clr-boot-manager update would reset default. Correct?

Correct. (`/etc/`**kernel**`/cmdline-removal.d`)

**Comment by intelkevinputnam** [comment]
@bwarden Thanks for the feedback. Doc is ready for final review.

**Comment by mvincerx** [comment]
LGTM. Any final feedback, reviewers?

File: source/clear-linux/tutorials/kernel-boot-params.rst [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+   man clr-boot-manager
+
+.. contents:: 
+    :local:
+    :depth: 1
+
+Updating the boot manager
+*************************
+
+The system must be updated and rebooted each time a changes is made
```
Comment (bwarden) [inline source/clear-linux/tutorials/kernel-boot-params.rst]:
```suggestion
The system must be updated and rebooted each time a change is made
```

**Comment by mvincerx** [comment]
@intelkevinputnam Did you plan to have this merged before 10 am?

**Comment by intelkevinputnam** [comment]
@mvincerx Nope. I wasn't trying to get it in before 10 AM. Interesting process getting the conflicts merged.

**Comment by intelkevinputnam** [comment]
@mvincerx Looking at the merge history and number of files affected after rebasing, I'm thinking it makes more sense to resubmit this PR. Thoughts?

**Comment by mvincerx** [comment]
@intelkevinputnam. Resubmit new PR with `kernel-boot-params` is best. 👍 
Especially given this is a new doc. Please link back to this PR to reference its history in the new PR.

**Comment by intelkevinputnam** [comment]
Closing as I have resubmitted as PR #707.

**Pull request closed without merging**